diff --git a/No_portal_storms - Steam 0.G/modinfo.json b/No_portal_storms - Steam 0.G/modinfo.json
index 3dfb380..620950b 100644
--- a/No_portal_storms - Steam 0.G/modinfo.json	
+++ b/No_portal_storms - Steam 0.G/modinfo.json	
@@ -4,9 +4,8 @@
     "id": "minimod_no_portal_storms",
     "name": "0.G. No Portal Storms",
     "authors": [ "Candlebury" ],
-    "maintainer": [ "Candlebury" ],
+    "maintainers": [ "Candlebury" ],
     "description": "Disables Portal Storms.  Some vanilla missions and other mod content might become inaccesible.",
-    "steam_id": 3145472516,
     "category": "rebalance",
     "dependencies": [ "dda" ]
   },
